core/domain/question_domain.py

```diff
@@ -1002,6 +1042,11 @@ def validate(self):
                 'Expected last updated to be a datetime, received %s' %
                 self.last_updated)
 
+        if not isinstance(self.misconception_ids, list):
```

Comment:
Should we validate the items of the list?

Reply:
Yep, I think we can check if the contents of the list are also strings. I'll add it.